core/src/test/java/com/graphhopper/routing/util/spatialrules/AreaIndexTest.java

```diff
@@ -177,6 +169,29 @@ else if (areas.isEmpty())
         System.out.println(counts);
     }
 
+    private AreaIndex<CustomArea> createCountryIndex() throws IOException {
+        // todo: should there be some reusable code to create 'the' object mapper?
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JtsModule());
+        List<CustomArea> customAreas;
+        try (BufferedReader reader = Files.newBufferedReader(Paths.get("files/spatialrules/countries.geojson"))) {
+            JsonFeatureCollection jsonFeatureCollection = objectMapper.readValue(reader, JsonFeatureCollection.class);
+            customAreas = jsonFeatureCollection.getFeatures().stream().map(CustomArea::fromJsonFeature).collect(Collectors.toList());
+        }
+        return new AreaIndex<>(customAreas);
+    }
+
+    @Test
+    public void testCountries() throws IOException {
+        AreaIndex<CustomArea> countryIndex = createCountryIndex();
+        // Berlin
+        assertEquals("Deutschland", countryIndex.query(52.5243700, 13.4105300).get(0).getProperties().get("name"));
```

Comment:
Looks like that the name is currently referring to the local name. IMHO we should probably prefer the english version or what do you think?

Reply:
Well for this test it does not matter which property we look at. Or do you want to change the name field in `countries.geojson`?

Reply:
This is more about the properties we want to keep in the `countries.json`

Reply:
Ah yeah did we decide which ones we will keep yet? Anyway, I'll change this test to `name:en` already.